### DISCUSSION TITLE
Welcome to react-native-executorch Discussions!

### DISCUSSION THREAD

**Discussion by darthez** [software-mansion/react-native-executorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DarkSorrow** [comment]
Hello,
I've been playing a bit with langChain lately to create some mini chat bot. I search the langchain community but didn't find any link to this project but i admit i'm not yet that familiar with the whole AI ecosystem and was wondering if there are any plan or integration of this library with langChain

**Reply by chmjkb** [reply]
Hi!
Thank you for your question. As of today, there is no integration with LangChain. We are looking forward to introduce some integrations with RAG frameworks and we think that LangChain is a great place to start, so stay tuned for future releases. Also, if we notice large community interest in that topic we will put more emphasis into this.

**Comment by joshkim81** [comment]
Hi, thanks for this amazing project! It's really exciting. Are all releases considered production stable?

**Reply by chmjkb** [reply]
Hi,
Thanks for the kind words, we are really glad you like it. Yes the releases are considered production-ready, but as the library is in early stage of development we might introduce some breaking changes in the future, so just something to keep in mind 😉
